flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.11";
+    systems.url = "github:nix-systems/default";
+  };
+
+  outputs = { self, nixpkgs, systems, ... } @ inputs:
+    let
+      forEachSystem = nixpkgs.lib.genAttrs (import systems);
+    in {
+      packages = forEachSystem (system:
+        let
+            pkgs = nixpkgs.legacyPackages.${system};
+        in rec {
+          bobaserver-assets = pkgs.yarn2nix-moretea.mkYarnPackage {
+            name="boba-server";
+            version="0.0.1";
+            src = ./.;
+            dontFixup = true;
+            doDist = false;
+            buildPhase = ''
+              yarn build
+            '';
+            distPhase = "";
```

Comment:
I don't think we need this since `doDist = false;` is set? Will need to check and see.